pkg/security/ebpf/manager.go

```diff
@@ -53,10 +55,15 @@ func NewDefaultOptions() manager.Options {
 }
 
 // NewRuntimeSecurityManager returns a new instance of the runtime security module manager
-func NewRuntimeSecurityManager() *manager.Manager {
-	return &manager.Manager{
-		Probes:   probes.AllProbes(),
-		Maps:     probes.AllMaps(),
-		PerfMaps: probes.AllPerfMaps(),
+func NewRuntimeSecurityManager(supportsRingBuffers bool) *manager.Manager {
+	manager := &manager.Manager{
+		Probes: probes.AllProbes(),
+		Maps:   probes.AllMaps(),
+	}
+	if supportsRingBuffers {
```

Comment:
Maybe we should have a config option to force the usage of the perf map, even if we support the ring buffer ?

Reply:
Done